.circleci/config.yml

```diff
@@ -74,11 +72,23 @@ jobs:
       - run: bazel run @yarn//:yarn test --script_path=yarn_test.sh
       - run: xvfb-run -a ./yarn_test.sh
 
+  # Runs devserver benchmark test.
+  devserver_benchmark_test:
+    <<: *job_defaults
+    steps:
+      - checkout:
+          <<: *post_checkout
+      - restore_cache:
+          key: *cache_key
+      - run: bazel run @yarn//:yarn
+      - run: $(bazel info bazel-bin)/external/yarn/yarn benchmark-test
```

Comment:
Not sure why but calling `bazel run @yarn//:yarn benchmark-test` hangs

Reply:
https://github.com/alexeagle/angular-bazel-example/wiki#setup

Reply:
`bazel run --script_path=benchmark_test.sh @yarn//:yarn benchmark-test && ./benchmark_test.sh`